lib/barbeque/message_handler/job_retry.rb

```diff
@@ -16,8 +16,11 @@ def initialize(message:, job_queue:)
       end
 
       def run
-        job_retry = Barbeque::JobRetry.find_or_initialize_by(message_id: @message.id)
-        job_retry.update!(job_execution: job_execution)
+        begin
+          job_retry = Barbeque::JobRetry.create(message_id: @message.id, job_execution: job_execution)
```

Comment:
Retry in barbeque means re-enqueuing a message having `type: "JobRetry"` to SQS. Thus it can be possible.

If we don't dedup this, it can be possible that barbeque-console says it is retried once but actually it was retried twice. In that situation, log for one of two retries will be lost.